src/index.js

```diff
@@ -74,6 +74,15 @@ VirtualMachine.prototype.stopAll = function () {
 };
 
 /**
+ * Clear out current running project data.
+ */
+VirtualMachine.prototype.clear = function () {
+    this.runtime.dispose();
+    this.editingTarget = null;
+    this.emitTargetsUpdate();
```

Comment:
I could be convinced that this is the responsibility of the `vm.clear()` caller.